gprofiler/profilers/java.py

```diff
@@ -274,6 +276,37 @@ def get_ap_version() -> str:
 T = TypeVar("T", bound="AsyncProfiledProcess")
 
 
+class ProcessUptimeCalculator:
+    DEFAULT_BINS = [5, 10, 30, 60, 300]
+
+    def __init__(self, process: Process):
+        self._create_time: Optional[float]
+        self._process = process
+        try:
+            self._create_time = process.create_time()
+        except psutil.NoSuchProcess:
+            self._create_time = None
+
+    def get_alive_time(self) -> Optional[float]:
+        if self._create_time is None:
+            return None
+        return time.time() - self._create_time
+
+    def get_alive_time_bin(self, bins: List[int] = None) -> Optional[str]:
+        alive_time = self.get_alive_time()
+        if alive_time is None:
+            return None
+
+        if not bins:  # None or an empty list -> use default
+            bins = self.DEFAULT_BINS
+        prev_bin_time = 0
+        for bin_time in bins:
+            if bin_time >= alive_time:
+                return f"run_time: {prev_bin_time}-{bin_time} seconds"
+            prev_bin_time = bin_time
+        return f"run_time: longer than {prev_bin_time} seconds"
```

Comment:
```suggestion
        return f"run_time: {prev_bin_time}+ seconds"
```
Matter of taste but I think it'll be easier to read